app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -142,7 +142,7 @@ def process_collection_item(item, key)
     end
 
     def map_labels(model_name, labels)
-      ContainerLabelTagMapping.map_labels(@label_tag_mapping, model_name, labels)
+      ResourceLabelTagMapping.map_labels(@label_tag_mapping, model_name, labels)
     end
 
```

Comment:
@bronaghs It certainly can be. The immediate plan is to add a `providers` column, and then refactor the code from there.